### PR TITLE
fix bug with new release branch naming

### PR DESCRIPTION
**Pull request by nikita-b**

### CODE REVIEW

**Review by ademidoff — DISMISSED** [review]

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1911-d312e48
client docker - perconalab/pmm-client-fb:PR-1911-d312e48
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1911-d312e48.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-1911-d312e48&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1911-d312e48.tar.gz

**Comment by JNKPercona** [comment]
API tests have failed, Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/7489/

**Comment by JNKPercona** [comment]
pmm2-client testsuite has failed, Please check: BATS: https://pmm.cd.percona.com/job/pmm2-testsuite/6907/

**Comment by JNKPercona** [comment]
UI tests have failed, Please check: UI: https://pmm.cd.percona.com/job/pmm2-ui-tests/9316/

File: build/bin/vars [hunk 1]
```diff
@@ -9,10 +9,17 @@ tmp_dir=${root_dir}/tmp
 # * pmm_version to contain only MAJOR.MINOR.PATCH: '2.0.0';
 # * pmm_release to contain only pre-release part (may be empty): 'alpha3'.
 pmm_branch=$(git rev-parse --abbrev-ref HEAD)
-full_pmm_version=$(cat ${root_dir}/VERSION)
-if [[ ${pmm_branch} != "release"* && ${pmm_branch} != "PMM-2.0" && ${pmm_branch} != "main" ]]; then
-    full_pmm_version=${full_pmm_version}-${pmm_branch}-$(git rev-parse --short HEAD)
+
+full_pmm_version=${full_pmm_version}-${pmm_branch}-$(git rev-parse --short HEAD)
+
+# TODO Maybe it makes sense to use variable from job here
```
Comment (denisok) [inline build/bin/vars:15]: it does

File: build/bin/vars [hunk 2]
```diff
@@ -9,10 +9,17 @@ tmp_dir=${root_dir}/tmp
...
+if [[ ${pmm_branch} =~ release-* || \
+      ${pmm_branch} =~ pmm-2.* || \
+      ${pmm_branch} == "main" || \
```
Comment (denisok) [inline build/bin/vars]: main and PMM-2.0 is covered above `full_pmm_version=${full_pmm_version}-${pmm_branch}-$(git rev-parse --short HEAD` ?

**Review by denisok — APPROVED** [review]

File: build/bin/vars (hunk 2 above)
Comment (nikita-b) [reply build/bin/vars]: Yep, you're right. We need only `pmm-2` and `release-` here

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1911-459d3f5
client docker - perconalab/pmm-client-fb:PR-1911-459d3f5
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1911-459d3f5.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-1911-459d3f5&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1911-459d3f5.tar.gz

**Comment by JNKPercona** [comment]
API tests have failed, Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/7492/

**Comment by JNKPercona** [comment]
pmm2-client testsuite has failed, Please check: BATS: https://pmm.cd.percona.com/job/pmm2-testsuite/6910/

**Comment by JNKPercona** [comment]
UI tests have failed, Please check: UI: https://pmm.cd.percona.com/job/pmm2-ui-tests/9322/

**Pull request merged**